sirepo/pkcli/service.py

```diff
@@ -67,41 +70,68 @@ def _handle_signals(signums):
         finally:
             [signal.signal(x[0], x[1]) for x in o]
 
+    def _safe_kill_process(proc):
```

Comment:
`_safe_kill` goes after `_kill`. We store methods, functions, and classes alphabetically unless they can't be sorted that way.